src/components/LawProposals.vue

```diff
@@ -80,4 +85,22 @@ export default defineComponent({
     }
   }
 }
+
+.sitOutButton {
+  position: absolute;
+  background: white;
+  padding: 4pt;
+  border: 1px solid;
+  border-radius: 6pt;
+  transform: translate3d(250px, 350px, 400px);
+  transition-duration: 0.1s;
+}
+
+.sitOutButton:hover {
+  background: lightgrey;
+}
+
+.sitOutButton:active {
+  transform: translate3d(250px, 350px, 400px) scale(1.1);
+}
```

Comment:
How about showing the button only if the laws are lifted up?

Reply:
Nice idea. Right now, i (we?) just need the functionality to do more testing of the game. Details of the presentation are important - but in my opinion not now.